docs/data/about/teamMembers.json

```diff
@@ -315,5 +315,13 @@
     "about": "Amateur cook, DIYer, and film photographer",
     "github": "KenanYusuf",
     "twitter": "KebabYusuf"
+  },
+  {
+    "name": "Nikita Ossei",
+    "title": "Customer Support",
+    "location": "London, United Kingdom",
+    "locationCountry": "uk",
```

Comment:
This needs to be to work:

```suggestion
    "locationCountry": "gb",
```

Right now, it shows:

<img width="590" alt="SCR-20240602-cbod" src="https://github.com/mui/material-ui/assets/3165635/990b6a68-ec44-46fb-8a8b-181c25e08e88">

fixed in 50414c9f924831d9b081ab48d2b5e7a41aebce80 (pushed to `master` and `next`)